src/en/03_user_guide.md

```diff
@@ -1,23 +1,156 @@
+
+**Monitoring dashboard:** The monitoring tabs display graphs with the following information:
+
+- Transaction Rate
+- Backends
+- Autovacuum Workers Usage
+- WAL Production Rate
+- Temp Files Creation Rate
+- Autovacuum Freeze
+- Blocked Locks
+- Database Size
+- Database Growth Rate
+- Heap Cache Miss Ratio
+- Index Cache Miss Ratio
+- Seq Scan Ratio
+- Long Transaction
+- Long Query
+- Long Autovacuum
+- Checkpoints
+- Bloat: Top 20 Tables
+- In Recovery
+- Autovac Freeze: Top 20 Tables
+- Database Size
+- Backends
+- Activity
+
+The number of graphs displayed can be controlled by clicking on `Manage units` and selecting the desired graphs. Personalized monitor units may be created by `Manage units → new unit`. This will open a new tab where the name, type, refresh interval, and template can be specified. Once a template has been selected, you may edit the script on the text editor beneath it.
+
+![Display of the monitor unit creation form](./images/monitor_unit.png)
+
+**Backends:** The backends tab displays a table with information about ports, addresses, events, and database activity.
+
+---
 
 ## Snippets
 
-## Command history
+The snippets panel can be accessed by clicking on the snippet symbol on the right side menu. You may write code on the text editor and save it. The indentation feature will automatically rewrite the SQL in a pattern.
+
+![Image displaying the snippets panel](./images/snippet.png)
+
+To use a saved snippet, open a query tap and right click on the text editor. Here you may use, overwrite, or create a snippet.
+
+![Image showing how to use snippets on the query tab](./images/snippet1.png)
+
+---
+
+## Command History
+
+The command history is available in the query and console tabs. It displays a list of previously executed queries/commands in a given time range. To copy a query from the history, right click and select `Copy Content To Query Tab`.
+
+![Image showing the query history](./images/history.png)
+
+---
 
 ## Explain
 
-## Code autocompletion
+The **Explain** and **Explain Analyze** features provide a visualization of the PostgreSQL execution plan for a query. The Explain feature provides the visualization with cost information while the Explain Analyze feature provides extra information -- such as time, number of rows filtered, and buffers. 
+
+You may access the graph metrics by clicking on `Settings` at the top right. This allows to display information about cost, time, and rows. Next, you may click on a graph node to display more information about it. Finally, zoom in and out of the graph with the scroll wheel.
+
+![Screenshot for the explain analyze feature](./images/explain_analyze.png)
+
+---
+
+## Code Autocompletion
+
+To aid at typing speed, the application will suggest tables or indexes to autocomplete the query.
+
+![Image showing the code autocompletion](./images/autocomplete.png)
+
+---
+
+## PostgreSQL Configuration Management
+
```

Comment:
Maybe we should add more general information about this functionality? Basically this is a visual interface about ALTER SYSTEM: https://www.postgresql.org/docs/current/sql-altersystem.html